MUICore/setup.py

```diff
@@ -39,7 +39,7 @@ def run(self):
     version="0.0.1",
     packages=find_packages(exclude=["tests", "tests.*"]),
     install_requires=[
-        "manticore @ git+https://github.com/trailofbits/manticore.git@chess",
+        "manticore @ git+https://github.com/trailofbits/manticore.git@634b6a4cdc295c93027b1dbe5037e574cf76200b",
```

Comment:
I like that this is a commit (makes it much easier to reproduce and also know what works) instead of a branch!

It'd be nice to have a comment describing that this commit is following the upstream `chess` branch that has various fixes not yet merged to `master`